examples/vision/meta_mnist.py

```diff
@@ -134,11 +136,22 @@ def main(lr=0.005, maml_lr=0.01, iterations=1000, ways=5, shots=1, tps=32, fas=5
 
     use_cuda = not args.no_cuda and torch.cuda.is_available()
 
-    torch.manual_seed(args.seed)
     random.seed(args.seed)
+    np.random.seed(args.seed)
+    torch.manual_seed(args.seed)
+    if use_cuda:
+        torch.cuda.manual_seed(args.seed)
+    torch.backends.cudnn.deterministic = True
```

Comment:
I believe cudnn is not used when the device is cpu. So looks like even line 144 and line 145 can move inside the `if use_cuda`.